recommender/static/html/recommender.html

```diff
@@ -129,109 +128,102 @@
                 </label>
                 <input type="button" value="Endorse resource" class="recommender_endorseResource"/>
             </div>
             <div class="recommender_flagResourcePage" style='display: none;' aria-hidden="true">
                 <div class="recommender_modifyPageTitle"></div>
-                <label>Reason&nbsp;
+                <label>${_("Reason")}&nbsp;
                     <input type="text"
                         class="recommender_flagReason" 
                         name="flagRationale"
-                        placeholder="Reason for why this resource should be flagged"/>
+                        placeholder="${_("Reason for why this resource should be flagged")}"/>
                 </label>
                 <input type="button" value="Flag resource" class="recommender_flagReasonSubmit"/>
             </div>
             <div class="recommender_editResourcePage" style='display: none;' aria-hidden="true">
                 <div class="recommender_modifyPageTitle"></div>
-                <label>Title&nbsp;<span class='recommender_redTxt' aria-hidden="true"></span>
+                <label>${_("Title")}&nbsp;<span class='recommender_redTxt' aria-hidden="true"></span>
                     <textarea
-                        aria-label='Provide a resource title, required'
+                        aria-label='${_("Provide a resource title, required")}'
                         class="recommender_editTitle"
-                        placeholder="Provide a meaningful title so other students know whether this is useful to them"></textarea>
+                        placeholder="${_("Provide a meaningful title so other students know whether this is useful to them")}"></textarea>
                 </label>
-                <label>Location&nbsp;<span class='recommender_redTxt' aria-hidden="true"></span>
+                <label>${_("Location")}&nbsp;<span class='recommender_redTxt' aria-hidden="true"></span>
                     <input type="text"
-                        aria-label='Provide a resource location in url, required'
+                        aria-label='${_("Provide a resource location in url, required")}'
                         class="recommender_editUrl"
-                        placeholder="http://en.wikipedia.org/wiki/Edx"/>
+                        placeholder="http://en.wikipedia.org/wiki/edX"/>
                 </label>
-                <label>Description&nbsp;
+                <label>${_("Description")}&nbsp;
                     <textarea
                         aria-label='Provide a resource description'
                         class="recommender_editDescriptionText"
-                        placeholder="Provide a meaningful description so other students know whether this is useful to them"></textarea>
+                        placeholder="${_("Provide a meaningful description so other students know whether this is useful to them")}"></textarea>
```

Comment:
@OmarIthawi it renders fine, but I found that the translation is broken due to double-quotation sequence I guess, I'll continue testing and be back with a fix

Reply:
sorry, my bad. Broken in my tutor. Looks like I need some caffeine  😅

Reply:
I've pushed the double-qutation fix, @OmarIthawi

Reply:
Thanks! done.